qcfractal/interface/models/common_models.py

```diff
@@ -35,6 +34,7 @@ class DriverEnum(str, Enum):
     energy = 'energy'
     gradient = 'gradient'
     hessian = 'hessian'
+    properties = 'properties'
```

Comment:
always wondered about the necessity of an extra "properties" option. is it mostly for psi, where run through a different fn, or do you generally regard it as a separate calc type?

Reply:
Not entirely sure, I view that as more experimental and we will see how it evolves.